doc/source/data/aggregations.rst

```diff
@@ -0,0 +1,148 @@
+.. _aggregations:
+
+==============
+Aggregations
+==============
+
+Ray Data provides a flexible and performant API for performing aggregations on :class:`~ray.data.dataset.Dataset`. 
+Internally, aggregations are powered by the :ref:`hash-shuffle backend <hash-shuffle>`.
+
+.. note::
+    To use the hash-shuffle algorithm for aggregations, you need to set the shuffle strategy explicitly:
+    
+    .. testcode::
+        
+        from ray.data.context import DataContext, ShuffleStrategy
+        
+        # Set hash-shuffle as the shuffle strategy
+        DataContext.get_current().shuffle_strategy = ShuffleStrategy.HASH_SHUFFLE
```

Comment:
Add a note referring to a Richard's blog on comparing range-based vs shuffle-based one (we'll update once i fix it)